thread_safe_set.go

```diff
@@ -0,0 +1,31 @@
+package movavg
+
+import "sync"
+
+// ThreadSafe synchronizes access to Moving Average calculator.
+func ThreadSafeSet(ma_set MASet) MASet {
```

Comment:
Could you, please, make it camelCase (the `ma_set` arg name)? snake_case seems way too foreign for Go.

(actually, it can/should be renamed to just `set` - see my comment about `Set`/`NewSet`)